### PR TITLE
Safety moth

### PR DESCRIPTION
**Pull request by SkyCinnamon**
## About the Pull Request

Old safety moth posters from the old codebase that are applicable to the current setting / some modified ones

original [PRs](https://github.com/BeeStation/BeeStation-Hornet/pull/1747) from [Bee](https://github.com/BeeStation/BeeStation-Hornet/pull/1982)
## Why It's Good For The Game

Saftey moth was a staple that many of us couldnt live without from the old codebase. More variety of posters that arent awful or outdated references that look like they are from an erp server at a glance.

## Did you test it?

Hell yeah brother!
image

## Changelog

:cl: Sky
rscadd: safety moth posters made by Azlanon and AyyRobotics
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 1]
```diff
@@ -293,3 +293,40 @@
...
 	name = "John Fleet poster"
 	desc = "A poster produced by the SCGF. It depicts Petty Officer John Fleet, a character used to boost troop morale, standing in front of the SCG flag proudly. Nobody knows if he really existed."
+
+///TeguStation posters
+
+/// Moth poster sprites are created by Azlanon and AyyRobotics
+/decl/poster/tegu_1
+	icon_state ="tposter1"
+	name = "Safety Moth - Poisoning"
+	desc = "This informational poster uses Safety Moth(TM) to ask the viewer not to poison the station donuts."
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]:
```suggestion
	desc = "This informational poster uses Safety Moth(TM) to ask the viewer not to poison the ship donuts."
```

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 2]
```diff
@@ -293,3 +293,40 @@
...
+
+/decl/poster/tegu_2
+	icon_state ="tposter2"
+	name = "Safety Moth - Hardhats"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to wear hardhats at all times. It's like a lamp for your head!"
+/decl/poster/tegu_3
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]:
```suggestion
	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to wear hardhats at all times. It's like a lamp for your head!"
	
/decl/poster/tegu_3
```

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 3]
```diff
@@ -293,3 +293,40 @@
...
+	icon_state ="tposter3"
+	name = "Safety Moth - Smokey?"
+	desc = "Only YOU can prevent plasma fires."
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]: Seems a bit odd that this is the only one that doesn't say "This information poster uses Safety Moth(TM)."

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 4]
```diff
@@ -293,3 +293,40 @@
...
+
+/decl/poster/tegu_4
+	icon_state ="tposter4"
+	name = "Safety Moth - Piping"
+	desc = "This informational poster uses Safety Moth(TM) to advise atmospheric technicians on the efficient usage of pipes. Proper pipe placement prevents poor performance!"
+
+/decl/poster/tegu_5
+	icon_state ="tposter5"
+	name = "Safety Moth - Supermatter"
+	desc = "This informational poster uses Safety Moth(TM) to promote proper safety equipment when working near a Supermatter Crystal."
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]: Supermatter crystal is not a proper noun.

```suggestion
	desc = "This informational poster uses Safety Moth(TM) to promote proper safety equipment when working near a supermatter crystal."
```

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 5]
```diff
@@ -293,3 +293,40 @@
...
+
+/decl/poster/tegu_6
+	icon_state ="tposter6"
+	name= "Safety Moth - Mindbreaker"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to seek CMO approval before making Mindbreaker. You shouldn't even be doing this."
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]:
```suggestion
	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to seek CMO approval before making mindbreaker toxin. You shouldn't even be doing this."
```

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 6]
```diff
@@ -293,3 +293,40 @@
...
+	desc = "This informational poster uses Safety Moth(TM) to advise atmospheric technicians on the efficient usage of pipes. Proper pipe placement prevents poor performance!"
+
+/decl/poster/tegu_5
+	icon_state ="tposter5"
+	name = "Safety Moth - Supermatter"
+	desc = "This informational poster uses Safety Moth(TM) to promote proper safety equipment when working near a Supermatter Crystal."
+
+/decl/poster/tegu_6
+	icon_state ="tposter6"
+	name= "Safety Moth - Mindbreaker"
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm]:
```suggestion
	name = "Safety Moth - Mindbreaker"
```

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 7]
```diff
@@ -293,3 +293,40 @@
 	icon_state="bsposter60"
 	name = "John Fleet poster"
 	desc = "A poster produced by the SCGF. It depicts Petty Officer John Fleet, a character used to boost troop morale, standing in front of the SCG flag proudly. Nobody knows if he really existed."
+
+///TeguStation posters
+
+/// Moth poster sprites are created by Azlanon and AyyRobotics
```
Comment (Shadowtail117) [inline code/game/objects/effects/decals/posters/bs12.dm:299]: @Cupax3 please clarify if modularization comments are needed

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/posters/bs12.dm (hunk 3 above)
Comment (SkyCinnamon) [reply code/game/objects/effects/decals/posters/bs12.dm]: > Seems a bit odd that this is the only one that doesn't say "This information poster uses Safety Moth(TM)."

as for this one thats what the original poster says and I like it more

Comment (Ilysen) [reply code/game/objects/effects/decals/posters/bs12.dm]: Plasma is also called phoron on baycode, so make sure you change that as well.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
It also occurred to me that unicode exists. You can replace `(TM)` with `™`.

File: code/game/objects/effects/decals/posters/bs12.dm (hunk 7 above)
Comment (Cupax3) [reply code/game/objects/effects/decals/posters/bs12.dm:299]: Nice to have, not required

File: code/game/objects/effects/decals/posters/bs12.dm [hunk 8]
```diff
@@ -293,3 +293,41 @@
...
+/decl/poster/tegu_1
+	icon_state ="tposter1"
+	name = "Safety Moth - Poisoning"
```
Comment (Cupax3) [inline code/game/objects/effects/decals/posters/bs12.dm:302]: To future-proof, please change Moth to Alvani. We can tease them with the secret lore:tm:

**Review by Cupax3 — COMMENTED** [review]
See reviews by Shadow and Ava, also some comments by me.

**Review by Ilysen — DISMISSED** [review]
Alvani don't exist right now, but I guess it doesn't hurt. /shrug

**Comment by SkyCinnamon** [comment]
> Alvani don't exist right now, but I guess it doesn't hurt. /shrug

im not opposed to keeping it as safety moth, im just following cupas suggestion.
Id probably rather keep it as safety moth and use it as a fun character Sol uses in propaganda or something as I doubt the moth on poster looks like the Alvani /shrug

**Comment by Ilysen** [comment]
That's definitely not how I envision an alvani. 😛 
Alrighty. Go ahead and change it back to moth, and if cupa wants to overrule me, then listen to them.

**Review by Cupax3 — DISMISSED** [review]
LGTM, now squish

**Review by Cupax3 — APPROVED** [review]
You made me approve twice hrng

**Pull request merged**